### PR TITLE
SCRUM-3605 serialize sgdStrainBackground, add to ForPublic view

### PR DESCRIPTION
**Pull request by cmpich**

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/model/entities/GeneDiseaseAnnotation.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public class GeneDiseaseAnnotation extends DiseaseAnnotation {
 	@IndexedEmbedded(includeDepth = 1)
 	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
 	@ManyToOne
-	@JsonView({ View.FieldsOnly.class })
+	@JsonView({ View.FieldsOnly.class, View.ForPublic.class })
```
Comment (devopsagr) [inline src/main/java/org/alliancegenome/curation_api/model/entities/GeneDiseaseAnnotation.java:49]: @cmpich this is not going to work unless its on production.

**Review by devopsagr — CHANGES_REQUESTED** [review]

File: src/main/java/org/alliancegenome/curation_api/model/entities/GeneDiseaseAnnotation.java (hunk 1 above)
Comment (devopsagr) [reply src/main/java/org/alliancegenome/curation_api/model/entities/GeneDiseaseAnnotation.java:49]: Sorry logged in at the wrong user. (oblodgett)

Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/model/entities/GeneDiseaseAnnotation.java:49]: @cmpich this is not going to work unless its on production.

**Review by oblodgett — CHANGES_REQUESTED** [review]

**Comment by cmpich** [comment]
I also will create a new release branch with this and a corresponding PR for production. Will still need this change on alpha, though.

**Comment by oblodgett** [comment]
Changes put in production get rolled back to alpha there is no need for this PR then.

**Pull request closed without merging**